compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/Types.java

```diff
@@ -1096,6 +1096,18 @@ private boolean isTupleTypeAssignable(BType source, BType target, Set<TypePair>
         return true;
     }
 
+    private boolean checkAllTupleTypeMembersBelongNoType(BTupleType tupleType) {
+        for (BType memberType : tupleType.tupleTypes) {
+            switch (memberType.tag) {
+                case TypeTags.NONE:
+                    return true;
```

Comment:
The method name says check**All** but we seem to return on the first match of noType, is this correct?